discover/controller.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2017 Istio Authors
```

Comment:
given all of the previous Istio logic has been removed, this is very generic k8s now, this could be removed.

Reply:
done